modules/util/source-formatter/src/main/java/com/liferay/source/formatter/check/UpgradeSetResultsSetTotalMethodCheck.java

```diff
@@ -43,6 +43,31 @@ protected String[] getValidExtensions() {
 		return new String[] {"java", "jsp", "jspf"};
 	}
 
+	private String _addTryCatch(
+		String newContent, String lastMethodCall, String stringBuilder) {
```

Comment:
`lastMethodCall` could be only `methodCall`, and `setResultsAndTotalMethodCall` > `newMethodCall`